lib/src/base-styles/_nx-colors.scss

```diff
@@ -92,6 +92,21 @@
   --nx-color-counter: var(--nx-swatch-grey-05);
   --nx-color-counter-active-text: var(--nx-swatch-white);
 
+  //NxProgressBar
+  --nx-color-progress-bar-background: var(--nx-swatch-indigo-95);
+  --nx-color-progress-bar-progress: var(--nx-swatch-teal-60);
+  --nx-color-progress-bar-text-success: var(--nx-swatch-green-25);
```

Comment:
```suggestion
  --nx-color-progress-bar-text-success: var(--nx-color-validation-valid);
```

Reply:
> > I think it's worth reaching out to design to ask about the light mode counter here. In NxCounter, the text & border are `grey-05`, and in NxProgressBar are black. I wonder if that's intentional? That could clean up our variables a bit.
> 
> Or that in light & dark mode, the counter text color shifts from black to `indigo-05`-- perhaps that could also be amalgamated into one standard colour?

I think they need to stay as two variables. After switching to dark mode, the progress bar counter's text color and border are no longer the same color. The border becomes the same color as the counter's background.